tests/functionality/testRandGmres.cpp

```diff
@@ -113,10 +113,10 @@ int runTest(int argc, char *argv[])
 
   matrix_handler.setValuesChanged(true, memspace);
 
-  real_type tol = 1e-12; // iterative solver tolerance
+  real_type tol = constants::LOOSE_TOL;
```

Comment:
This is easier to understand:
```suggestion
  real_type tol = 1000*constants::MACHINE_EPSILON;
```